frontend/views/components/Page.vue

```diff
@@ -84,6 +84,22 @@ export default ({
     },
     checkIsTouch () {
       this.ephemeral.isTouch = window.innerWidth < DESKTOP
+    },
+    findAndScrollToAnchor (str) {
+      const anchorEl = this.$el.querySelector(`:is([id='${str}'], [name='${str}'], [href='#${str}'])`)
+      anchorEl && anchorEl.scrollIntoView({ behavior: 'smooth', block: 'start' })
+    }
+  },
+  watch: {
+    '$route': {
+      immediate: true,
+      handler (to, from) {
+        if (to.hash && from?.hash !== to.hash) {
+          setTimeout(() => {
+            this.findAndScrollToAnchor(to.hash.slice(1))
+          }, 100)
+        }
+      }
```

Comment:
Scrolling down to a specific anchor point via specifying `#string` is a browser's native behavior. But for some reason, this does not work reliably in our app(I've observed this in some other SPA websites at the other company) so we need a logic to handle this manually. Since `Page.vue` is used in all pages, decided this is the good place for it.